Makefile

```diff
@@ -0,0 +1,51 @@
+SHELL := '/bin/bash'
+SENDGRID := $(shell aws-vault exec lpa-dev -- aws secretsmanager get-secret-value --secret-id development/opg_lpa_front_email_sendgrid_api_key | jq -r .'SecretString')
+GOVPAY := $(shell aws-vault exec lpa-dev -- aws secretsmanager get-secret-value --secret-id development/opg_lpa_front_gov_pay_key | jq -r .'SecretString')
+ORDNANCESURVEY := $(shell aws-vault exec lpa-dev -- aws secretsmanager get-secret-value --secret-id development/opg_lpa_front_ordnance_survey_license_key | jq -r .'SecretString')
+
+.PHONY: all
+all:
+	@${MAKE} dc-up
+
+.PHONY: dc-run
+dc-run:
+	export OPG_LPA_FRONT_EMAIL_SENDGRID_API_KEY=${SENDGRID}; \
```

Comment:
exports shown here need to have std out redirected to `/dev/null`

Reply:
ok so i found makefiles behave differently, i'll add an `@` to the relevant bits